src/lib/y2storage/inhibitors/mdadm_auto_assembly.rb

```diff
@@ -0,0 +1,49 @@
+# encoding: utf-8
+
+# Copyright (c) 2018 SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast2/execute"
+
+module Y2Storage
+  # class to inhibit mdadm from doing auto assembly
+  class MdadmAutoAssembly
+    include Yast::Logger
+
+    def inhibit
+      log.info "set udev ANACONDA property"
+
+      begin
+        Yast::Execute.locally!("/sbin/udevadm", "control", "--propertyx=ANACONDA=yes")
```

Comment:
Is it really `propertyx`? There is `property` value used in the `uninhibit` method below...

Reply:
Ok, that was indeed a test to check error handling.